generators/app/templates/src/style.less

```diff
@@ -7,5 +7,8 @@
 
   //to build your css, run `lessc src/style.css dist/style.css --clean-css`
   // in your terminal
-
+	.rootContainer {
```

Comment:
Why is this present? It's probably okay, I'm just curious :grinning:

Reply:
It's just to give an example and have something look good at the start.
Empty file doesn't look good I think 😅